comment/templates/comment/child_comment.html

```diff
@@ -1,5 +1,5 @@
 
-<blockquote class="js-child-comment blockquote text-wrap mb-4">
+<div class="js-child-comment blockquote text-wrap mb-4">
     <hr/>
     {% include 'comment/image.html' with comment=comment %}
-</blockquote>
```

Comment:
As far as I can see the [documentation](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/blockquote), I believe the blockquote element was misplaced here.

Reply:
This was used to get the current comment style, I believe I got this block from bootstrap. However, feel free to update the style and the templates.